docs/source/contributor-guide/roadmap.md

```diff
@@ -43,3 +43,84 @@ start a conversation using a github issue or the
 make review efficient and avoid surprises.
 
 [The current list of `EPIC`s can be found here](https://github.com/apache/datafusion/issues?q=is%3Aissue+is%3Aopen+epic).
+
+# Quarterly Roadmap
+
+A quarterly roadmap will be published to give the DataFusion community
```

Comment:
I slightly reworded this introduction section; Otherwise the content is the same

Reply:
That is an excellent question. I started a discussion here https://github.com/apache/datafusion/issues/11442 to see what we can find out